entity-test-lib/src/main/java/org/terracotta/entity/PassthroughEndpoint.java

```diff
@@ -157,11 +155,11 @@ private void checkEndpointOpen() {
   }
 
   public long getCurrentId() {
-    return idGenerator.get();
+    return 0L;
```

Comment:
Isn't this a potential behavior change?

Reply:
No.  nothing ever changed it.  constructor value was zero